docs/customer-swagger.yaml

```diff
@@ -779,6 +779,83 @@ components:
             - $ref: '#/components/schemas/NullRightsRetentionStrategy'
             - $ref: '#/components/schemas/RightsRetentionStrategy'
             - $ref: '#/components/schemas/OverridableRightsRetentionStrategy'
+        allowFileUploadFor:
+          type: array
+          items:
+            type: string
+            enum:
+              - ACADEMIC_ARTICLE
+              - ACADEMIC_CHAPTER
```

Comment:
These should now follow naming pattern in enum: AcademicChapter, AcademicLitteratureReview and so on.